HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,302 @@
+  https://github.com/hashgraph/solo
+
+As shown, the Hedera ecosystem already contains several sub-projects that must be part of `PROJECT`.
+All those sub-projects are necessary to deploy and run a full Hashgraph-based network and interact with that network.
+All mentioned projects are currently part of the Hashgraph organization at GitHub.
+Supplemental repositories include supporting tooling to deploy and validate a network built using the above codebases.
+In addition to the aforementioned repositories and projects, the community and ecosystem around Hedera and its unique
+hashgraph consensus algorithm is quite extensive, and we will want to transition a number of third-party projects
+coming from the community as an initial step once `PROJECT` has been set up. 
+An example of such a community project is https://github.com/OpenElements/hedera-enterprise.
+
+## GitHub issues
+The Hedera projects use GitHub issues today.
+Together, the projects contain more than 1,000 open issues that contain information that should be preserved since they
+contain great contributions from the community. Based on that, we propose to migrate the issues to the newly
+created repositories.
+
+## GitHub Actions 
+Next to the issues, each of the repositories uses GitHub Actions for continuous integration.
+While the GitHub Action workflow definitions are stored as part of the repository, we use custom action runners on
+hardware in the Google Cloud.
+The default GitHub action runners can not be used due to the complexity of `PROJECT`, especially for the services sub-project.
+To continue the development of `PROJECT` based on best practices and continuous integration, we propose to set up
+clones of the current hardware and action runners.
+The Hedera Hashgraph projects define teams with specific authorities.
+Those configurations should be migrated, and maintainers and committers for each sub-project should be defined.
+The complete list of GitHub users that should be added as maintainers or committers to the
+sub-projects will be submitted later.
+
+## Documentation
+The Hedera project provides a repository for the documentation of the Hedera network, including its deployment and
+utilization of ever-evolving services.
+That repository can be found at https://github.com/hashgraph/hedera-docs.
+The content of the repository uses gitbook to create complete documentation that is currently hosted
+at https://docs.hedera.com/hedera.
+We propose to create a neutral hosted version of the documentation at https://docs.PROJECT.com, removing or modifying
+existing content that is Hedera specific.
+We will work on that directly after the setup to remove all those parts from the transited repository.
+
+## DCO
+We suggest setting up a DCO process by LFX for all `PROJECT` repositories.
+The DCO documents will be based on the LF templates.
+We will provide the final DCO documents separately.
```

Comment:
I don't understand this statement. DCO is provided using the `-s` option on the `git commit` command. If we are transferring repos, we will need to determine how to meet DCO requirements.

Reply:
We currently use the same GitHub plugin for DCO checks as LF. We will check since what date the plugin is in use and how to handle commits that happened before we used the plugin. I will update the paragraph.